content/en/docs/concepts/architecture/cri.md

```diff
@@ -47,4 +47,4 @@ requires a restart of the kubelet.
 
 ## {{% heading "whatsnext" %}}
 
-- Learn more about the CRI [protocol definition](https://github.com/kubernetes/cri-api/blob/c75ef5b/pkg/apis/runtime/v1/api.proto)
+- Learn more about the CRI [protocol definition](https://github.com/kubernetes/cri-api/tree/master)
```

Comment:
let's reference the file: https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1/api.proto

Reply:
would you like me to add this link? instead of the link which i have added!

Reply:
We're cautious about linking to _master_. Can we link to a tag instead? You can use a shortcode like `{{< skew currentVersion >}}` (which right now renders as `1.31`) if that helps. Or, link to a specific fixed release tag - it might go stale, but it's unlikely to turn into a broken hyperlink.

Reply:
linking latest via the tag sounds like the best idea!